SmartEVSE-3/src/evse.cpp

```diff
@@ -5031,22 +5093,40 @@ void setup() {
 
     CP_ON;           // CP signal ACTIVE
 
-
+    //mongoose
+    mg_mgr_init(&mgr);  // Initialise event manager
 }
 
 void loop() {
-    //this loop is for non-time critical stuff that needs to run approx 1 / second
-    vTaskDelay(1000 / portTICK_PERIOD_MS);
-    getLocalTime(&timeinfo, 1000U);
-    if (!LocalTimeSet && WIFImode == 1) {
-        _LOG_A("Time not synced with NTP yet.\n");
+
+    static unsigned long lastNtpCheck = 0;
```

Comment:
Why is this an improvement to the vTaskDelay 1000 statement? Its one more static var ...

Reply:
Relates to the suggested change above of putting the networking related functionality on the main task. Then we need the loop function to execute as frequent as possible and can't put delays on it anymore.